ui/app/mainui/AppMain.qml

```diff
@@ -803,20 +803,16 @@ Item {
                         if (activeSectionType === Constants.appSection.chat)
                             return Constants.appViewStackIndex.chat
                         if (activeSectionType === Constants.appSection.community) {
-
-                            for(let i = this.children.length - 1; i >=0; i--)
-                            {
+                            for (let i = this.children.length - 1; i >=0; i--) {
                                 var obj = this.children[i]
-                                if (obj && obj.sectionId && obj.sectionId === appMain.rootStore.mainModuleInst.activeSection.id)
-                                {
-                                    obj.active = true
+                                if (obj && obj.sectionId && obj.sectionId === appMain.rootStore.mainModuleInst.activeSection.id) {
                                     return i
                                 }
                             }
 
                             // Should never be here, correct index must be returned from the for loop above
-                            console.error("Wrong section type: ", appMain.rootStore.mainModuleInst.activeSection.sectionType,
-                                        " or section id: ", appMain.rootStore.mainModuleInst.activeSection.id)
+                            console.error("Wrong section type:", appMain.rootStore.mainModuleInst.activeSection.sectionType,
+                                          "or section id: ", appMain.rootStore.mainModuleInst.activeSection.id)
```

Comment:
are we sure this space is not needed before the "or"?

Reply:
Yeah, the comma `,` takes care of that